sqldelight-sample/src/main/java/com/example/sqldelight/hockey/ui/MainActivity.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.sqldelight.hockey.ui
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import com.example.sqldelight.hockey.R
+
+class MainActivity : Activity() {
+  private val listView by lazy { findViewById<Button>(R.id.list) }
+  private val teamsView by lazy { findViewById<Button>(R.id.teams) }
```

Comment:
locals and no lazy? or you don't even need locals just `findViewById().setOnClick...`

Reply:
you use it everywhere else. it kills me but whevs